server/index.js

```diff
@@ -115,14 +116,22 @@ async function handleAlert(location, alertType) {
       }
       braveAlerter.startAlertSession(alertInfo)
     } else {
+      currentSession.numberOfAlerts += 1 // increase the number of alerts for this session
+      // boolean value of whether the client should be allowed to request this sensor be reset
+      const allowResetRequest = currentSession.numberOfAlerts >= helpers.getEnvVar('SESSION_MIN_NUMBER_OF_ALERTS_TO_ALLOW_RESET_REQUEST')
+
       db.saveSession(currentSession, pgClient) // update updatedAt
 
       braveAlerter.sendAlertSessionUpdate(
         currentSession.id,
         client.repsonderPushId,
         client.responderPhoneNumbers,
         location.phoneNumber,
-        t('alertAdditionalAlert', { lng: client.language, alertTypeDisplayName, deviceDisplayName: location.displayName }),
+        t(allowResetRequest ? 'alertAdditionalAlertWithResetChoice' : 'alertAdditionalAlert', {
```

Comment:
Looks like we haven't got any test files for this `handleAlert` function yet. It would be great for you to add one, but it's possible that it's not there because it was difficult to write. You could add a Task in ClickUp to remind us to add it later if it looks like it's going to be really complicated.

Reply:
I'm going to work on writing a test file for this function. Might take a while, I want to be thorough :)

Reply:
Cool. This might be a good time to start moving functions out of `index.js` and into more usefully-named files. Often this can make them more testable too because they will separate the initialization (which will remain in index.js) with the testable implementation.

Reply:
Yeah, smart! I'll go ahead and do that, since Buttons does something similar for its rak_button_press route.